README.md

```diff
@@ -368,6 +412,26 @@ Type: `Array<Function>`. Default: [Autoprefixer](https://github.com/postcss/auto
 All of the CSS you load via [`stylesheets`](#stylesheets) is run through [PostCSS](http://postcss.org/), so you can apply any [PostCSS plugins](https://github.com/postcss/postcss/blob/master/docs/plugins.md) to it.
 By default, only [Autoprefixer](https://github.com/postcss/autoprefixer) is applied.
 
+### siteBasePath
+
+Type: `string`. Default: `''`.
+
+Root-relative path to the base directory on the domain where the site will be deployed.
+
+There's a good chance your app isn't at the root of your domain. So this option represents the path of your site *within* that domain.
+
+For example, if your app is at `https://www.special.com/ketchup/*`, you should set `siteBasePath: ketchup`.
+
+### stylesheets
+
+Type: `Array<string>`. Absolute paths, please. Default: `[]`.
+
+An array of filenames pointing to stylesheets that you want to include in your site.
+
+These will be processed by PostCSS with [Autoprefixer](https://github.com/postcss/autoprefixer) and any other [postcssPlugins](#postcssplugins) that you specify; concatenated in the order specified; and added to the `<head>` of your document.
```

Comment:
Currently they are not automatically added to the `<head>`, right?

Reply:
Yup, will remove that part.